### DISCUSSION TITLE
version 1.0.1

### DISCUSSION THREAD

**Discussion by broisnischal** [neeswebservices/reactjs-image-zoom#6] [Announcements]
# Reactjs Image Zoom

This is the maintained release of the package and you can try it by downloading. from the npmjs package.  

`version 1.0.1`

If you face any problem while installing it discuss below. 

<hr /><em>This discussion was created from the release <a href='https://github.com/neeswebservices/reactjs-image-zoom/releases/tag/reactjs-stable-release'>version 1.0.1</a>.</em>

**Comment by Sudip-pandey** [comment]
This is awesome !! that i previewd the commit and the git repo !
